source/locales/pl.json

```diff
@@ -228,12 +228,12 @@
   "Chinese (Simplified)": "Chiński uproszczony",
   "Updated": "Zaktualizowane",
   "By": "przez",
-  "Donate Bitcoin": "Bitcoin wspomóż akcję",
+  "Donate Bitcoin": "Dobrowolna dotacja BitCoin",
```

Comment:
however, if be in accordance with original, can add: wspomóż akcję POPRZEZ BitCoin. Word order is now corrctly with Polish lang.